src/lift-state-manager.ts

```diff
@@ -18,5 +18,146 @@ export default class LiftStateManager extends EventEmitter<Events> {
     });
   }
 
+  static liftModeToString(liftMode: number): string {
+    switch (liftMode) {
+      case RomiCore.LiftState.MODE_AGV:
+        return 'AGV';
+      case RomiCore.LiftState.MODE_EMERGENCY:
+        return 'Emergency';
+      case RomiCore.LiftState.MODE_FIRE:
+        return 'Fire';
+      case RomiCore.LiftState.MODE_HUMAN:
+        return 'Human';
+      case RomiCore.LiftState.MODE_OFFLINE:
+        return 'Offline';
+      default:
+        return `Unknown (${liftMode})`;
+    }
+  }
+
+  static doorStateToString(doorState: number): string {
+    switch (doorState) {
+      case RomiCore.LiftState.DOOR_OPEN:
+        return 'Open';
+      case RomiCore.LiftState.DOOR_CLOSED:
+        return 'Closed';
+      case RomiCore.LiftState.DOOR_MOVING:
+        return 'Moving';
+      default:
+        return `Unknown (${doorState})`;
+    }
+  }
+
+  static motionStateToString(motionState: number): string {
+    switch (motionState) {
+      case RomiCore.LiftState.MOTION_DOWN:
+        return 'Down';
+      case RomiCore.LiftState.MOTION_STOPPED:
+        return 'Stopped';
+      case RomiCore.LiftState.MOTION_UP:
+        return 'Up';
+      default:
+        return `Unknown (${motionState})`;
+    }
+  }
+
   private _liftStates: Record<string, RomiCore.LiftState> = {};
 }
+
+export class LiftRequestManager {
+  static liftModes = [
+    RomiCore.LiftRequest.REQUEST_END_SESSION,
+    RomiCore.LiftRequest.REQUEST_AGV_MODE,
+    RomiCore.LiftRequest.REQUEST_HUMAN_MODE,
+  ];
+
+  static doorModes = [
```

Comment:
```suggestion
  static readonly liftModes = [
```
Doesn't look like it should be modified